integrated_channels/integrated_channel/management/commands/transmit_courseware_data.py

```diff
@@ -78,4 +78,14 @@ def send_data_task(username, channel_code, channel_pk):
         channel,
     )
 
-    channel.transmit_course_data(user)
+    # channel.transmit_course_data(user)
+    try:
+        channel.transmit_course_data(user)
```

Comment:
So I was digging around in this management command as part of the data passback work and noticed that the reason the entire job fails is because we are creating celery tasks to do work by channel, not by enterprise customer. So that is why if one enterprise customer fails, everything fails, because they are all being done in the same thread. I think we need to refactor to kick off a celery task per enterprise customer, which was the original intent from what I remember. Is there a reason doing these celery tasks by enterprise customer would be problematic @haikuginger and @douglashall?